.gitignore

```diff
@@ -189,13 +189,13 @@ src/include/pbs_version.h
 src/include/pbs_config.h
 src/include/pbs_config.h.in
 src/include/pbs_config.h.in~
-src/server/queue_attr_def.c
-src/server/resc_def_all.c
-src/server/resv_attr_def.c
-src/server/sched_attr_def.c
-src/server/svr_attr_def.c
-src/server/job_attr_def.c
-src/server/node_attr_def.c
+src/lib/Libattr/queue_attr_def.c
+src/lib/Libattr/resc_def_all.c
+src/lib/Libattr/resv_attr_def.c
+src/lib/Libattr/sched_attr_def.c
+src/lib/Libattr/svr_attr_def.c
+src/lib/Libattr/job_attr_def.c
+src/lib/Libattr/node_attr_def.c
```

Comment:
all the master attr def files were in the server. With is_attr() now using master attr defs, any program (qmgr for instance) that wanted to use is_attr() would have to link with the server code (which in turn would have a million dependencies of its own and it could become very messy). To avoid that, I thought it would make more sense to move the master attr defs to libattr so one could just link with libattr and use is_attr().